djstripe/stripe_objects.py

```diff
@@ -968,21 +1014,22 @@ def get_default_account(cls):
 
 class StripeCard(StripeSource):
     """
-You can store multiple cards on a customer in order to charge the customer later.
-(Source: https://stripe.com/docs/api/python#cards)
+    You can store multiple cards on a customer in order to charge the customer later.
```

Comment:
http://www.writersdigest.com/online-editor/which-vs-that

Think `which` works as the sentence `A model` still holds true, we are just saying `which represents a Stripe Card object` to give more context.